src/index.ts

```diff
@@ -31,54 +47,26 @@ async function main() {
       UserOverrideResolver,
       ...relationResolvers
     ],
+    authChecker: ({ root, args, context, info}) => {
+      // console.log(context.expressContext.req.userId, 'userId')
+      if(!context.expressContext.req.userId) {
+        return false;
+      } 
+      return true;
+    },
```

Comment:
Applies authChecker to make sure that the express middleware add the userId to the context req